internal/project/project.go

```diff
@@ -492,7 +492,7 @@ func getCodeBlocks(data []byte) (document.CodeBlocks, error) {
 	identityResolver := identity.NewResolver(identity.DefaultLifecycleIdentity)
 	d := document.New(data, identityResolver)
 
-	if f, err := d.Frontmatter(); err == nil && f != nil && f.Runme.Session.ID != "" {
+	if f, err := d.Frontmatter(); err == nil && f != nil && !f.Runme.IsEmpty() && f.Runme.Session != nil && f.Runme.Session.ID != "" {
```

Comment:
What do you think about encapsulating this condition  `!f.Runme.IsEmpty() && f.Runme.Session != nil && f.Runme.Session.ID != ""` into a method `func (m *RunmeMetadata) IsFooBarBaz()`?